src/main/java/redis/clients/jedis/params/ZRangeParams.java

```diff
@@ -0,0 +1,108 @@
+package redis.clients.jedis.params;
+
+import static redis.clients.jedis.Protocol.Keyword.BYLEX;
+import static redis.clients.jedis.Protocol.Keyword.BYSCORE;
+import static redis.clients.jedis.Protocol.Keyword.LIMIT;
+import static redis.clients.jedis.Protocol.Keyword.REV;
+import static redis.clients.jedis.args.RawableFactory.from;
+
+import redis.clients.jedis.CommandArguments;
+import redis.clients.jedis.Protocol.Keyword;
+import redis.clients.jedis.args.Rawable;
+
+public class ZRangeParams implements IParams {
+
+  private final Keyword by;
+  private final Rawable min;
+  private final Rawable max;
+  private boolean rev = false;
+
+  private boolean limit = false;
+  private int offset;
+  private int count;
+
+  private ZRangeParams() {
```

Comment:
Why we need a `private and empty args` constructor?

Reply:
@yangbodong22011 It's not uncommon to use Jedis with reflection. I've added this contributor so that to reduce issues when used with reflection because two of mandatory arguments of zrange command are included in this class. Our other params classes are fine because they don't contain mandatory arguments.

Reply:
@sazzad16 ok, thanks for your explain.